src/CommandLineParser.ts

```diff
@@ -162,6 +172,39 @@ function inputConflicts(videoUrls: Array<string | number> | undefined,
 }
 
 
+function isOutputTemplateValid(argv: any): boolean {
+    let finalTemplate: string = argv.outputTemplate;
+    const elementRegEx = RegExp(/{(.*?)}/g);
+    let match = elementRegEx.exec(finalTemplate);
+
+    // if no template elements this fails
+    if (match) {
+        // keep iterating untill we find no more elements
+        while (match) {
+            if (!templateElements.includes(match[1])) {
+                logger.error(
+                    `'${match[0]}' is not aviable as a template element \n` +
+                    'Please check the template elements aviable in the README \n',
+                    { fatal: true }
+                );
+
+                process.exit(1);
+            }
+            match = elementRegEx.exec(finalTemplate);
+        }
+    }
+    // bad template from user, switching to default
+    else {
+        logger.warn('Empty output template provided, using default one');
+        argv.outputTemplate = '{title} - {date} {uniqueID}';
```

Comment:
There is no `{date}`, you probably wanted `publishDate` here?

```
.\destreamer.ps1 -i https://VIDEO_URL -t ""

[FATAL ERROR] '{date}' is not aviable as a template element
Please check the template elements aviable in the README
```

Reply:
Yeah the default was there as POC and I forgot to change it

Reply:
✅